### PR TITLE
Added Tom Clancy's Rainbow Six 3

### PR DESCRIPTION
**Pull request by ImperatorS79**

### CODE REVIEW

**Comment by madoar** [comment]
Just asking:
Why is this script in testing?

**Comment by ImperatorS79** [comment]
Because it's the wrong version of the script ;)

**Comment by madoar** [comment]
What about the steam script? It is also set to testing^^

File: Applications/Games/Tom Clancy's Rainbow Six 3 : Raven Shield/Local (1.0->1.6)/script.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        }
+            
+        new Downloader()
+            .wizard(wizard)
+            .url(url)
+            .checksum(sha1)
+            .to(originDirectory + exeName)
+            .get();
+            
+        wine.run(originDirectory + exeName,"");
```
Comment (madoar) [inline Applications/Games/Tom Clancy's Rainbow Six 3 : Raven Shield/Local (1.0->1.6)/script.js]: I think you can run this command without the last argument `""`.

**Pull request merged**